Tests/Storage/FileServiceTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace EMS\CoreBundle\Tests\Service;
+
+use EMS\CommonBundle\Storage\Service\StorageInterface;
+use EMS\CoreBundle\Service\FileService;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use function file_put_contents;
+use function hash_file;
+use function hash_init;
+use function printf;
+use function strlen;
+use function unlink;
+
+
+class FileServiceTest extends WebTestCase
+{
+    public function testStorageServices()
+    {
+        self::bootKernel();
+
+        $this->assertNotNull(self::$container);
+
+        /**@var FileService $fileService */
+        $fileService = self::$container->get('ems.service.file');
+
+        /**@var StorageInterface $storage*/
+        foreach ($fileService->getStorages() as $storage)
+        {
+            $this->assertNotNull($storage);
+
+            $this->verifyStorageService($storage);
+        }
+
+    }
+
+    private function verifyStorageService(StorageInterface $storage)
+    {
+
+        $this->assertTrue($storage->health());
+
+        printf("\n%s\n", $storage->__toString());
```

Comment:
Won't this will pollute output during testing?

Reply:
Fixed 66a616628122fb7ac222fd13070eea38dea9d75e